merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertConstraintRunsAction.java

```diff
@@ -0,0 +1,55 @@
+package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.constraints.model.Violation;
+import gov.nasa.jpl.aerie.merlin.server.models.Constraint;
+import gov.nasa.jpl.aerie.merlin.server.models.PlanId;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Map;
+
+import static gov.nasa.jpl.aerie.constraints.json.ConstraintParsers.violationP;
+
+/* package local */ class InsertConstraintRunsAction implements AutoCloseable {
+  private static final @Language("SQL") String sql = """
+    insert into constraint_run (constraint_id, constraint_definition, plan_id, simulation_dataset_id, status, violations)
+    values (?, ?, ?, ?, ?::constraint_status, ?::json)
+  """;
```

Comment:
```suggestion
  private static final @Language("SQL") String sql = """
    insert into constraint_run (constraint_id, constraint_definition, plan_id, simulation_dataset_id, violations)
    values (?, ?, ?, ?, ?::json)
  """;
```

Reply:
Yeah I missed this, it's all fixed up now!

Reply:
Did you push this change? I'm not seeing it.

Reply:
Yeah I did!
https://github.com/NASA-AMMOS/aerie/blob/ecdb34191f4e1093ff3c662cd65da36412381923/merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertConstraintRunsAction.java#L19